src/hooks/useThemeProps/index.ts

```diff
@@ -0,0 +1,13 @@
+import {useContext, createContext} from 'react';
+import * as Modifiers from '../../commons/modifiers';
+import {ThemeManager} from 'style';
+
+const EmptyContext = createContext({});
+
+const useThemeProps = (props: any, componentName: string) => {
+  const themeContext = ThemeManager.getThemeContext();
+  const context = useContext(themeContext ?? EmptyContext);
```

Comment:
I would expect context to have a method `getThemeProps`, instead of using external function `Modifiers.getThemeProps`